apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/ModelTests.cs

```diff
@@ -154,5 +154,38 @@ public async Task GetModelAsync()
 
             Assert.Equal(modelId, model.Reference.ModelId);
         }
+
+        [Fact]
+        public void ModelExtractJob()
```

Comment:
Rename this and next test to CreateModelExtractJob{Async}, to match the methods they're testing?